src/index.html

```diff
@@ -16,17 +19,33 @@
       href="style.css"
     />
   </head>
-  <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
+  <body class="search-bar__body">
+    <form
+      class="search-bar__form search-bar__form--big"
+      action="http://localhost:8080/"
+      data-qa="big"
```

Comment:
Remember to use 2 spaces for indentation. The attributes of the `form` tag are not aligned properly. Each attribute should be on its own line with proper indentation.